addon-test-support/@ember/test-helpers/dom/fire-event.js

```diff
@@ -1,5 +1,15 @@
 import { merge } from '@ember/polyfills';
 
+// eslint-disable-next-line require-jsdoc
+function checkMouseEventExistence() {
+  try {
```

Comment:
can you memoize this?

Reply:
Do you have an example of where else Memoization it is done in this repo? Or another example I can emulate?

Reply:
Something like:

```js

const MOUSE_EVENT_CONSTRUCTOR = (() => {
  try {
    new MouseEvent(‘test’);
    return true;
  } catch(e) {
    return false;
  }
})();